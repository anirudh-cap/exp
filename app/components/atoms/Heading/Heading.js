import React from 'react';
import PropTypes from 'prop-types';
import classnames from 'classnames';
import withStyles from 'utils/withStyles';
import styles from './style';

export const Heading = ({ className, children }) => (
  <h2 className={classnames(styles.heading, className)}>{children}</h2>
);

Heading.propTypes = {
  children: PropTypes.node,
  className: PropTypes.string,
};

Heading.defaultProps = {
  className: '',
};

export default withStyles(Heading, styles);
