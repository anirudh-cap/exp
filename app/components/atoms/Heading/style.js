import { css } from 'styled-components';

export default css`
  font-size: 1.5em;
  color: #000;
`;
