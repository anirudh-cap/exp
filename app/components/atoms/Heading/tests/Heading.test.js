import React from 'react';
import { shallow } from 'enzyme';
import { Heading } from '../Heading';

describe('<Heading />', () => {
  it('should render the Heading', () => {
    const renderedComponent = shallow(<Heading className="">header</Heading>);
    expect(renderedComponent).toMatchSnapshot();
  });
});
