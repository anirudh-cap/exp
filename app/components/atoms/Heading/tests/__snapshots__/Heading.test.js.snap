// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> should render the Heading 1`] = `
<h2
  className=""
>
  header
</h2>
`;
